addons/website_slides/models/slide_channel.py

```diff
@@ -133,7 +133,8 @@ def _recompute_completion(self):
 
             if not record.channel_id.active:
                 continue
-            elif not was_finished and record.completed_slides_count >= record.channel_id.total_slides:
+            # Add record to completed unless there are no slides in the channel (can happen when unlinking)
```

Comment:
This is straightforward to read and will be explained in the commit message, I don't think it needs a comment here.

Reply:
ok